documentation/src/main/asciidoc/topics/con_cache_store_sql_composites.adoc

```diff
@@ -0,0 +1,54 @@
+[id='sql-store-composite-keys-values_{context}']
+= Composite keys and values
+
+You can use SQL stores with database tables that contain composite primary keys or composite values.
+
+To use composite keys or values, you must provide {brandname} with Protobuf schema that describe the data types.
+You must also add `schema` configuration to your SQL store and specify the message names for keys and values.
+
+[TIP]
+====
+{brandname} recommends annotating your classes and generating Protobuf schema with the ProtoStream processor.
+You can then easily upload your Protobuf schema for remote caches through the {brandname} Console, CLI, or REST API.
+For more information, see the _Cache encoding and marshalling_ documentation.
```

Comment:
Yes, it is linked at the bottom of the topic in the `Additional resources` section. It's best practice to put links at the end because inline linking risks getting users off task. So the link is there, just at the end.